Assets/MRTK/Providers/WindowsMixedReality/XRSDK/WindowsMixedRealityDeviceManager.cs

```diff
@@ -181,13 +285,29 @@ protected override SupportedControllerType GetCurrentControllerType(InputDevice
 
             if (inputDevice.characteristics.HasFlag(InputDeviceCharacteristics.Controller))
             {
+#if HP_CONTROLLER_ENABLED
```

Comment:
Ah missed that this pattern is used here as well. Along the lines of https://github.com/microsoft/MixedRealityToolkit-Unity/pull/8794#discussion_r530647822. Also, i think we've found that the `isHPController = trackedMotionControllerStates.ContainsKey(controllerId)` assumption doesn't hold, so it should probably just check `!hasTouchpad`